src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -76,6 +80,9 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
 
   void enableCleanupOfHelmReleaseOnSuccess() {
     config.clearHelmReleaseOnSuccess = true;
+
+        WarningCollector.addPipelineWarning("Helm-ReleaseonSuccess-deprication", "`enableCleanupOfHelmReleaseOnSuccess` is now the default, please remove it from your `Jenkinsfile`.", LocalDate.of(2023, 8, 31));
```

Comment:
deprication is still spelt wrong even though I spelt it right in my suggestion

Reply:
updated typo